en-US/Design.xml

```diff
@@ -637,12 +637,12 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
           </listitem>
           <listitem>
             <para>
-              As much as possible, leave the system-wide default as <varname>become: false</varname> or <varname>become: no</varname> and if a single task needs privileges, set <varname>become: true</varname> or <varname>become: yes</varname> on that task.
+              As much as possible, leave the system-wide default as <varname>become: false</varname> and if a single task needs privileges, set <varname>become: true</varname> on that task.
```

Comment:
Ideally, I'd like for us to specify "default what"? I'm guessing "default setting" (maybe?), but again - default setting for what? 

However, if we're scoping this PR to align the examples to only using `true` and `false`, then we can approve the PR as is and clarify these explanations at a later date.